command/var_put.go

```diff
@@ -581,10 +581,12 @@ func formatInvalidVarKeyChars(invalid []string) string {
 
 	// Sort the characters for output
 	charList := make([]string, 0, chars.Size())
-	for _, k := range chars.List() {
+	chars.ForEach(func(k string) bool {
```

Comment:
I'm ok with this but it just looks somewhat non-idiomatic

Reply:
Agreed, but it's either this or creating a copy of the data which is worse. The Go team is working behind the scenes to make iteration over user defined structs a thing, but who knows about the ETA

https://github.com/golang/go/discussions/54245
https://github.com/golang/go/discussions/56413